src/services/account/useAccountDetails.ts

```diff
@@ -117,9 +117,9 @@ export const PlanSchema = z
   .object({
     baseUnitPrice: z.number(),
     benefits: z.array(z.string()),
-    billingRate: z.string().nullable(),
+    billingRate: z.nativeEnum(BillingRate).nullish(),
     marketingName: z.string(),
-    monthlyUploadLimit: z.number().nullish(),
+    monthlyUploadLimit: z.number().nullable(),
```

Comment:
this is actually nullable() vs. nullish(), just matching all the other schema types too